tests/acceptance/features/coreApiWebdavMove1/moveFolderToExcludedDirectory.feature

```diff
@@ -12,7 +12,6 @@ Feature: users cannot move (rename) a folder to or into an excluded directory
   Scenario Outline: Rename a folder to an excluded directory name
     Given using <dav_version> DAV path
     And user "Alice" has created folder "/testshare"
-    And the administrator has updated system config key "excluded_directories" with value '[".github"]' and type "json"
```

Comment:
Similar comment here - these excluded-directory scenarios are not complete without specifying settings.